webextensions/api/declarativeNetRequest.json

```diff
@@ -0,0 +1,1656 @@
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": "15"
+              },
+              "safari_ios": "mirror"
+            }
+          },
+          "beacon": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "csp_report": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "84"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "font": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "84"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "image": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "84"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "imageset": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
```

Comment:
`imageset` is not supported by Safari, it is covered by `image`. A Note would be good.

Reply:
```suggestion
                  "version_added": false,
                  "notes": "Use <code>image</code> instead."
```